### PR TITLE
Registration route

### PR DESCRIPTION
**Pull request by matheisson**
Test pull request

### CODE REVIEW

**Comment by krs89** [comment]
I guess this comment goes under the whole pull request. There must be an option to comment the code itself, line by line. Let's find it....

**Comment by matheisson** [comment]
You were right. Lets merge the branches!

File: app.py [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+# @app.route('/applicants')
+# def app_list():
+#     applicants = Applicant.select()
+#     return render_template('VALAMI.HTML', applicants=applicants)
 
+@app.route('/registration', methods=['POST', 'GET'])
+def registration():
+    if request.method == 'POST':
+        Applicant.create(first_name = request.form["frist_name"], last_name= request.form["last_name"], hometown= request.form["hometown"], application_code= request.form["application_code"],
+                            status= "In Progress", application_time = datetime.now())
```
Comment (krs89) [inline app.py]: this "In Progress" looks weird in the database

File: app.py [hunk 2]
```diff
@@ -17,24 +17,30 @@ def main_page():
...
 
-@app.route('/registration', methods=['POST', 'GET'])
+
+@app.route('/registration/', methods=['POST', 'GET'])
 def registration():
     if request.method == 'POST':
-        new_applicant = Applicant.create(first_name = request.form["first_name"], last_name= request.form["last_name"], hometown= request.form["hometown"],
-                            status= ["In Progress"], application_time = datetime.now())
-        #flash("Registration Successful!")
+        new_applicant = Applicant.create(
```
Comment (matheisson) [inline app.py:24]: Looks a lot more better like this!

File: app.py [hunk 3]
```diff
@@ -17,24 +17,30 @@ def main_page():
...
+            first_name=request.form["first_name"],
+            last_name=request.form["last_name"],
+            hometown=request.form["hometown"],
+            status=["In Progress"],
```
Comment (krs89) [inline app.py:28]: sooo this 'in progress' looks weird in the database, it goes in in a bad format

**Pull request merged**